views/partials/prs.hbs

```diff
@@ -40,7 +40,7 @@
                             submitted a pull request
                             <a class="blue link underline-hover" href="{{ this.url }}">{{ this.repo_name }}#{{ this.number }}</a>
                         </div>
-                        <div class="light-silver">{{ this.title }}</div>
+                        <div class="light-silver">{{ this.title }} at {{ this.created_at }}</div>
```

Comment:
As we're showing just the date and not the time, can we change 'at' to 'on' ?

Reply:
Sure yea i guess 'on' looks much better.